contact.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar:
+nav: "docs"
+permalink: /contact.html
+layout: splash
+header:  
+    video: /assets/images/waves.mp4
+    overlay_color: "rgba(63, 108, 152, 1)"
```

Comment:
I've noticed that we're repeating the `overlay_color` in every page. It should use a variable.